include/ofi_epoll.h

```diff
@@ -0,0 +1,133 @@
+/*
+ * Copyright (c) 2018 Intel Corporation.  All rights reserved.
```

Comment:
You need to copy the copyrights from the original file, or at least pull over all that are relevant for this code.

Reply:
ofi_signal.h has only intel copyright from 2011 to 2015. Should I make this 2011-2018 here?

Reply:
I *think* it would need to be 2011-2015, 2018.  But I'm not an expert in this area.